src/test/resources/selenide.properties

```diff
@@ -0,0 +1,35 @@
+# Selenide configuration properties
+
+# Default browser (chrome, firefox, edge, etc.)
+selenide.browser=chrome
+
+# Default language (vi, en)
+selenide.language=vi
+
+# Run mode: local or grid
+selenide.mode=local
+
+# Browser window size (width x height)
+selenide.browserSize=1920x1080
+
+# Base URL
+selenide.baseUrl.VjAir=https://www.vietjetair.com/
```

Comment:
Actually, selenide prefix key is required by Selenide config only. If you listing your own config, no need to add "selenide." prefix